gui/elements/_layer_list.py

```diff
@@ -190,6 +200,7 @@ def pop(self, index=-1):
             Removed item.
         """
         item = self._list.pop(index)
+        self._qt.remove_layer(item)
```

Comment:
This should be hooked on to the `remove` event instead of called directly. There are some examples at the bottom of this file iirc.

Reply:
ok nice, was wondering where to put it. similarly for the `_add` and `_reorder` event I assume? I'll give that a try.